### ISSUE TITLE
Java on Heroku Dynos?

### ISSUE THREAD

**Issue by caseywatts** [jkutner/heroku-buildpack-minecraft#9]
How does this buildpack end up getting `java` to be available on the dyno?

I get this error after setting up and deploying:
```
2017-06-02T22:08:53.455874+00:00 app[web.1]: bin/minecraft: line 44: java: command not found
```

Notes:
- ngrok stuff all seems to be working great~
- the only buildpack on this repo is the default one `https://github.com/kr/heroku-buildpack-inline.git`

**Comment by ShayBox** [comment]
Not a very good place to ask for support, project is basically dead, im surprised I was just googling and happened to click issues and saw a post from 12 hours ago

**Comment by caseywatts** [comment]
I'm not sure I'd call this project dead - It's been updated to the most recent Minecraft version even :)

Although maybe I should ping @jkutner on this?

**Comment by jkutner** [comment]
Project is definitely not dead. But the latest version of the Heroku stack no longer has the JVM installed by default. Someone needs to add this bit:

https://github.com/heroku/heroku-buildpack-jvm-common#usage-from-a-buildpack

Glad to accept a PR for it.

**Comment by jkutner** [comment]
Or could add the jvm-common buildpack to `app.json`

**Comment by jkutner** [comment]
Fixed by db2adc063ece198bd9ec94c626fb505a0801295c

**Comment by caseywatts** [comment]
you're the best @jkutner 🎉 
can't wait to try it :)
